Shared map state between modules

Kraci opened the pull request:
- route compute dummy module
- passing map state with SYMapView instance between modules
- option for pass only map state reference or make copy of map state (based on implementation in module)
- ModuleUIViewController class for abstract sdk init methods for modules using sdk
- Table View Controller addded to example app for showing multiple modules

Kraci committed:
Shared map state between modules

- route compute dummy module
- passing map state with SYMapView instance between modules
- option for pass only map state reference or make copy of map state (based on implementation in module)
- ModuleUIViewController class for abstract sdk init methods for modules using sdk
- Table View Controller addded to example app for showing multiple modules

jcaliksygic commented on SygicMapsKit/Classes/ModuleUIViewController.swift:
```diff
@@ -0,0 +1,22 @@
+import SygicMaps
+
+
+public class ModuleUIViewController: UIViewController {
```
premenoval by som to na SYMKModuleViewController

jcaliksygic commented on SygicMapsKit/Classes/Map/SYMKMapController.swift:
```diff
@@ -26,14 +26,11 @@ public class SYMKMapController: NSObject {
-        
-        mapView = SYMapView(frame: mapFrame, geoCenter: SYGeoCoordinate(latitude: 48.147, longitude: 17.101878)!, rotation: 0, zoom: 16, tilt: 0)
+    public init(with mapState: SYMKMapState, mapFrame: CGRect = .zero) {
+        mapView = mapState.loadMap()
```
tu by mal vstupovat atribut frame

Kraci committed:
Making copy of map state is now module independent

- pr naming fixes
- you can now decide if you want make copy of mapstate or use just reference, when initializing module
- MapState property is now property of SYMKModuleViewController class

Kraci committed:
File name fix
